File: Praktikum 5. Function/Soal 4/PRAK504-2310817310001-AvantioFierzaPatria.c
#include <stdio.h>

int reverse(int num)
{
    int reversedNum = 0;
    while (num > 0)
    {
        reversedNum = reversedNum * 10 + num % 10;
        num /= 10;
    }
    return reversedNum;
}

int main()
{
    int A, B;
    scanf("%d %d", &A, &B);

    A = reverse(A);
    B = reverse(B);

    int C = A + B;

    printf("%d", reverse(C));

    return 0;
}

File: Praktikum 5. Function/Soal 4/PRAK504-2310817310001-AvantioFierzaPatria.py
def reverse(num):
    reversed_num = 0
    while num > 0:
        reversed_num = reversed_num * 10 + num % 10
        num //= 10
    return reversed_num


A, B = map(int, input().split())

A_reversed = reverse(A)
B_reversed = reverse(B)

C = A_reversed + B_reversed

print(reverse(C))
